api/envoy/config/accesslog/v2/file.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
 package envoy.config.accesslog.v2;
+option java_outer_classname = "FileProto";
```

Comment:
Nit: I think a blank line between `package` and the `option` block would make sense here and elsewhere.